model/lib/account/account.dart

```diff
@@ -1,13 +1,30 @@
 import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:model/account/authentication_type.dart';
 
 class Account with EquatableMixin {
   final String id;
   final AuthenticationType authenticationType;
   final bool isSelected;
+  final AccountId? accountId;
+  final String? apiUrl;
```

Comment:
IMO, `jmapBaseUrl`, but I remembered we store it in `credentialRepository`. Any duplication?

Reply:
IMO,
- `jmapBaseUrl` should be user-secure. So I don't think it should be saved on `SharePreferences`.
- It will be duplicated with the `baseUrl` stored in `CredentialRepository`

WDYT? @hoangdat